skeleton-es2016-webpack/webpack.prod.config.js

```diff
@@ -35,11 +35,11 @@ module.exports = {
     loaders: [
       { test: /\.js$/, loader: 'babel', exclude: /node_modules/, query: {stage: 0} },
       { test: /\.css?$/, loader: 'style!css' },
-      { test: /\.html$/, loader: 'raw' },
-      { test: /\.(png|gif|jpg)$/, loader: 'url-loader?limit=8192' },
-      { test: /\.woff2(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: 'url-loader?limit=10000&minetype=application/font-woff2' },
-      { test: /\.woff(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: 'url-loader?limit=10000&minetype=application/font-woff' },
-      { test: /\.(ttf|eot|svg)(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: 'file-loader' }
+      { test: /\.html$/, loader: 'html' },
+      { test: /\.(png|gif|jpg)$/, loader: 'url?limit=8192' },
+      { test: /\.woff2(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: 'url?limit=10000&minetype=application/font-woff2' },
+      { test: /\.woff(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: 'url?limit=10000&minetype=application/font-woff' },
+      { test: /\.(ttf|eot|svg)(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: 'file' }
```

Comment:
Yeah, parts of the configs could be reused, but initially I didn't want to do that for keeping things simple, especially for people who are new to webpack.